components/Homepage/RoundActionCard/RoundActionCard.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  Center,
+  HStack,
+  Heading,
+  Spinner,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { Phase } from "services/PhaseMgmtService";
+
+interface Props {
+  phase: Phase;
+  roundId: number;
+  isAuthed?: boolean;
```

Comment:
nit - Idk why, but my gut would rather these optional user state params be required and coalesced at a higher level